packages/artifact/src/internal/utils.ts

```diff
@@ -65,7 +65,9 @@ export function isForbiddenStatusCode(statusCode?: number): boolean {
   return statusCode === HttpCodes.Forbidden
 }
 
-export function isRetryableStatusCode(statusCode?: number): boolean {
+export function isRetryableStatusCode(
+  statusCode?: number | undefined
```

Comment:
isn't `?` just sugar for `T | undefined`?

Reply:
Got rid of the `?`,